packages/dialtone-css/lib/build/less/components/stack.less

```diff
@@ -42,6 +42,36 @@
   }
 }
 
+.gap-options() {
+  &--100 {
+    .d-stack--gap-100();
+  }
+
+  &--200 {
+    .d-stack--gap-200();
+  }
+
+  &--300 {
+    .d-stack--gap-300();
+  }
+
+  &--400 {
+    .d-stack--gap-400();
+  }
+
+  &--450 {
+    .d-stack--gap-450();
+  }
+
+  &--500 {
+    .d-stack--gap-500();
+  }
+
+  &--600 {
+    .d-stack--gap-600();
+  }
+}
```

Comment:
I seem to remember we're aiming for shying away from mixins. I guess the precedent is already set for `DtStack`'s use of `.direction-options()`. 

I don't strictly have an opinion on it since I've lived and breathed mixins w/ SCSS and LESS, so I don't fault its use. Just wondering.

Reply:
Hm, it's not recommended to use here: https://github.com/dialpad/dialtone/blob/staging/packages/dialtone-vue3/.github/CONTRIBUTING.md#dialtone-usage
Maybe it's referring to some global mixins?

Reply:
> I seem to remember we're aiming for shying away from mixins. I guess the precedent is already set for `DtStack`'s use of `.direction-options()`.

This is only on the product side, within dialtone itself it's not a problem.

Reply:
It's a little bit confusing because it says "Please do not use any mixins or utility classes in Dialtone Vue components." but utilities are allowed on the consumer side